README.md

```diff
@@ -15,17 +15,9 @@ You can download the newest version of the wallet for each specific platform.
 - Download for [Linux (deb)](https://github.com/HathorNetwork/hathor-wallet/releases/download/v0.6.5-beta/hathor-wallet_0.6.5-beta_amd64.deb)
 - Download for [Linux (AppImage)](https://github.com/HathorNetwork/hathor-wallet/releases/download/v0.6.5-beta/Hathor.Wallet.0.6.5-beta.AppImage)
 
-### Warning Message
+### Warning Message for Windows:
 
-We are almost finishing the process of acquiring the certificates for macOS and Windows. While we don't get it you may see a warning message when opening the wallet.
-
-#### For macOS:
-
-![Warning macOS](https://drive.google.com/thumbnail?id=17gxGXmD1Zbd_uSsadnOem_i0tDR-Sdzh&sz=w500-h375)
-
-To be able to run you need to open Security & Privacy, click on the tab "General" and you will see a message "Hathor Wallet was blocked from opening because it is not from an identified developer.". So you just need to click on the button "Open Anyway" to use our wallet. For further information, see [macOS User Guide - Open a Mac app from an unidentified developer](https://support.apple.com/en-gb/guide/mac-help/mh40616/10.14/mac/10.14.3)
-
-#### For Windows:
+We are almost finishing the process of acquiring the certificates for Windows. While we don't get it you may see a warning message when opening the wallet.
```

Comment:
'we are almost finished with the process' or 'we're finishing the process'

Reply:
Updated, thanks